src/familiar/itemFamiliar.ts

```diff
@@ -12,7 +12,12 @@ export function bestFairy(): Familiar {
   if (!bestNonCheerleaderFairy) {
     const viableFairies = Familiar.all()
       .filter(
-        (f) => have(f) && findFairyMultiplier(f) && f !== $familiar`Steam-Powered Cheerleader`
+        (f) =>
+          have(f) &&
+          findFairyMultiplier(f) &&
+          f !== $familiar`Steam-Powered Cheerleader` &&
+          !f.physicalDamage &&
```

Comment:
just so you know i went through and did a runthrough of the familiars on the [wiki](https://kol.coldfront.net/thekolwiki/index.php/Familiars_by_type#Increases_item_drop) for item & damage and they all properly were tagged with their correct damage source, so this looks good.